.github/CODEOWNERS

```diff
@@ -1,6 +1,6 @@
 * @grafana/grafana-oncall-backend
 /grafana-plugin @grafana/grafana-oncall-frontend
-/docs @grafana/docs-gops
+/docs @grafana/docs-gops @grafana/grafana-oncall
```

Comment:
multiple groups listed acts as OR, not AND ([GH community discussion](https://github.com/orgs/community/discussions/22522#discussioncomment-3237152))